installation/install

```diff
@@ -279,12 +287,19 @@ script_header "INSTALLATION DU MAN DE GOOGLE"
 sudo cp $files/google.1 /usr/share/man/man1/google.1
 handle_error $?
 
-script_header "INSTALLATION DE TERMINATOR"
-sys_install terminator
+script_header "DESIREZ VOUS INSTALLER TERMINATOR ?"
+read -p "Installer terminator? (Un client console) [O/n]" yn
+case $yn in
+    [Nn]* );;
+    * ) script_header "INSTALLATION DE TERMINATOR" ; sys_install terminator;;
+esac
 
-script_header "INSTALLATION DES OUTILS COMPLEMENTAIRES"
-sys_install tree
-sys_install filezilla
+script_header "INSTALLATION DES OUTILS COMPLEMENTAIRES (TREE ET FILEZILLA)"
+read -p "Installer makefile-gen pour générer vos makefiles ? [O/n]" yn
```

Comment:
It's not makefile-gen that is installed.
Why put tree and filezilla in the same package ? it's not the same usage and is therefore pointless.
+ I don't want tree to be an optionnal package on this script for now